hack/docker-builder/entrypoint.sh

```diff
@@ -4,4 +4,12 @@ set -o pipefail
 
 source /etc/profile.d/gimme.sh
 export GOPATH="/root/go"
+
+# launch OVS
+function quit {
```

Comment:
i don't like the fact that we use docker-**builder** to run tests, but i guess we can live with it if it saves us from another container

Reply:
Yeah I don't know to create another container for that.